quizroyale/server/login_server.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+
+import jwt
+from flask import Flask, request, render_template, jsonify
+
+from quizroyale.db.db import DummyDbService
+from quizroyale.db.errors import NoSuchUserException
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    return render_template('login.html')
+
+
+@app.route('/login', methods=['POST'])
+def login():
+    db = DummyDbService()
```

Comment:
One instance of db should be enough, right now it's instantiated on every call to /login.

Reply:
What do you propose? put it free before methods, or use __init__?